contents/teams/brand-vibes/index.mdx

```diff
@@ -8,29 +8,29 @@ template: team
 
 ## Q2 2025 goals
 
-### Make PostHog a household name for engineers (Brian)
-- **Rationale:** Ads = Buzz = £££
-- **Things we could do:** Ads on Reddit. Video ads. Greater focus on AI products in ads. 
+### Ship the website (Cory & Eli)
+- **Rationale:** Better website = More vibes 
+- **Things we could do:** Finish, launch, monitor conversion, iterate.
+- **Planning issue:** Coming soon?
+- **We'll know we're successful when:** People love the new site and conversion doesn't tank long-term.
+
+### Make PostHog iconic (Lottie) 
+- **Rationale:** Instant recognition = A brand that sticks 
+- **Things we could do:** Scale design team. Creative direction for videos. Lead Do More Weird. 
+- **Planning issue:** Coming soon?
+- **We'll know we're successful when:** Design isn't a bottleneck, but quality doesn't suffer.
```

Comment:
'and more design work is shipped' maybe?

Reply:
That feels a little vague, because we'll always be shipping more design work (can't go backwards). If we want to add a hard metric or goal here, I think maybe the one to have is an average turnaround time or a statement about zero skipped jobs? But personally I'm happy to go with the vibe of it since this is so heavily taste dependent.